mmv1/products/alloydb/Cluster.yaml

```diff
@@ -452,3 +461,21 @@ properties:
       - !ruby/object:Api::Type::String
         name: 'sourceType'
         description: 'Type of migration source.'
+  - !ruby/object:Api::Type::Enum
+    name: clusterType
+    required: true
```

Comment:
Can we avoid adding this as required?

This will break existing customers. We should have a default PRIMARY if this field is needed.